.gitignore

```diff
@@ -54,3 +54,6 @@ vendor
 tilt.d
 tilt-settings.json
 .tiltbuild
+
+# User-supplied clusterctl hacks settings
+clusterctl-settings.json
```

Comment:
Since you're going to have to squash this before we lgtm it, could you please add a newline at the end of the file? (most editors/IDEs have a setting to do this automatically, if you want)

Reply:
done